### ISSUE TITLE
Error: Redefining          uint8/uint16/uint32/uint64 type which is part of ISPC language since          version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to detect that these types are defined. 

### ISSUE THREAD

**Issue by yurivict** [OpenImageDenoise/oidn#60]
Version 1.2.0
ispc-1.12.0.118
clang-9
FreeBSD 12.1

```
/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:10:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int8  uint8;
                 ^^^^

/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:11:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int16 uint16;
                 ^^^^^

/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:12:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int32 uint32;
                 ^^^^^

/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:13:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int64 uint64;
                 ^^^^^

--- local__core/output_reorder.dev.o ---
/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:10:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int8  uint8;
                 ^^^^

/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:11:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int16 uint16;
                 ^^^^^

/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:12:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int32 uint32;
                 ^^^^^

/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:13:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int64 uint64;
                 ^^^^^

--- local__core/upsample.dev.o ---
/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:10:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int8  uint8;
                 ^^^^
/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:11:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int16 uint16;
                 ^^^^^

/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:12:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int32 uint32;
                 ^^^^^

/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:13:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int64 uint64;
                 ^^^^^
--- local__core/color.dev.o ---
/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:10:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int8  uint8;
                 ^^^^

/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:11:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int16 uint16;
                 ^^^^^

/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:12:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int32 uint32;
                 ^^^^^

/usr/ports/graphics/oidn/work/oidn-1.2.0/core/math.ih:13:18: Error: Redefining 
        uint8/uint16/uint32/uint64 type which is part of ISPC language since 
        version 1.13. Remove this typedef or use ISPC_UINT_IS_DEFINED to 
        detect that these types are defined. 
typedef unsigned int64 uint64;
                 ^^^^^
```

**Comment by atafra** [comment]
ISPC 1.13 has not been released yet. Please use the latest stable version: 1.12. Thanks!

**Comment by atafra** [comment]
Or you could try the `devel` branch of OIDN, which now has the fix for this issue. But please note that we do not support non-stable versions of ISPC. Thanks!

**Comment by atafra** [comment]
Pushed fix to the `devel` branch.

**Comment by atafra** [comment]
We've released `v1.2.1`, which includes the fix.
